website/common/script/content/faq.js

```diff
@@ -1,6 +1,28 @@
+import forEach from 'lodash/forEach';
 import t from './translation';
 
-const NUMBER_OF_QUESTIONS = 12;
+const headings = [
+  'overview',
+  'set-up-tasks',
+  'sample-tasks',
+  'task-color',
+  'health',
+  'party-with-friends',
+  'pets-mounts',
+  'character-classes',
+  'blue-mana-bar',
+  'monsters-quests',
+  'gems',
+  'bugs-features',
+  'world-boss',
+  'group-plans',
+];
+
+const exclusions = {
+  android: [12, 13],
```

Comment:
maybe use the key in here - then we won't need to need to worry if we change the order someday

Reply:
A lot of what I've done here is exactly because we can't readily change the order--thanks to the translation strings being numbered 😅 But I can do that, anyway!

Reply:
What if we remove the numbers from the transactions at all ? then we could move things around how ever we want it and attach the numbers outside of the strings?

Reply:
oh the translation strings (keys) are numbered ok maybe then those need android /ios/web suffixes instead - then it might work out easier

Reply:
Changing the translation tokens would invalidate all the translations that have been done, though--I suppose if I rename them in all the different languages, that might work? 🤔

Reply:
I came up with another way that doesn't involve hacking the translated keys. Sending it up once I've run a functional test